includes/introduction.tex

```diff
@@ -8,30 +8,30 @@ \section{Introduction}
 real-world examples from a limited set of industrial
 applications~\cite{kaluza,kausler}. These test suites have helped
 developers identify implementation defects and develop more
-sophisticated solving heuristics. Unfortunately, as solvers grow,
-fixed benchmarks leave more functionality untested.
-There is an acute need for a more robust, inexpensive, and automatic way
+sophisticated solving heuristics. Unfortunately, as solvers grow
+fixed benchmarks remain stagnant, leaving increasing functionality untested.
+As such, there is an acute need for a more robust, inexpensive, and automatic way
 of generating benchmarks to test
 the correctness and performance of SMT solvers.
 
-Fuzzing is often used to test all kinds of software,
-including SAT solvers~\cite{fuzzsat}. Inspired by the utility of fuzzers,
-we introduce a fuzzer for string solvers, \fuzzer{}, and describe its value
+Fuzzing has been used to test all kinds of software
+including SAT solvers~\cite{fuzzsat}. Inspired by these fuzzers,
+we introduce \fuzzer{} and describe its value
 as an exploratory testing tool. We demonstrate its efficacy
-by presenting limitations it helped us find in
+by presenting limitations it helped discover in
 leading string solvers. To the best of our knowledge, \fuzzer{} is the
-first tool aimed at automatic generation of string constraints. \fuzzer{} can
+only tool aimed at automatic generation of string constraints. \fuzzer{} can
```

Comment:
Just curious: why "first" to "only"? "First" implies "only", but remains true as other ones are developed.

Reply:
First ages better, only is stronger for today (when we'll be reviewed).